spec/support/matchers/validation_helpers.rb

```diff
@@ -2,11 +2,11 @@ module ValidationHelpers
   private
 
   def check_errors(object, attribute, error)
-    !object.valid?(validation_context)
+    object.valid?(validation_context)
```

Comment:
Has this not changed what this line does altogether?

Reply:
This method does not care what is returned by object.valid?.  object.valid? is being called because it runs the validations and adds any errors to the object.